lua/obsidian/command.lua

```diff
@@ -95,7 +95,11 @@ command.open = function(client, data)
   local encoded_path = util.urlencode(tostring(path))
 
   local app = "/Applications/Obsidian.app"
-  if Path:new(app):exists() then
+  local sysname = vim.loop.os_uname().sysname
+  if sysname == "Linux" then
+    local cmd = ("xdg-open 'obsidian://open?vault=%s&file=%s'"):format(encoded_vault, encoded_path)
+    vim.fn.jobstart(cmd)
```

Comment:
Agreed, I just didn't want to mess with the mac-specific code since I don't have access to one for testing.

`jobstart` starts an os command asynchronously, so it doesn't block and doesn't display output should the command return something (for this you would need to attach something to the job later, or register a callback, but I think we don't care about potential output).

Reply:
Ah, I see. Well I feel like we *should* report when there's an error (we don't currently), in which case it sounds like it would be better to use the synchronous `os.execute()`. But that doesn't have to be in this PR, so I'm fine with using `vim.fn.jobstart()` for now.